website-custom-views/src/content/development/permissions.mdx

```diff
@@ -23,37 +23,26 @@ The `PERMISSIONS` variable contains a `View` and `Manage` properties:
 
 You can then use the `PERMISSIONS` variable to reference the permission in the application code.
 
-<Info>
-
-This feature is available from version `21.21.0` onwards.
+## With addtional permissions groups
 
 When using [additional permission groups](/concepts/oauth-scopes-and-user-permissions#permission-groups), you must also provide the unique group to the `resolveCustomViewResourceAccesses` function to generate the correct permission keys.
 
 The group names can be exported and referenced also in the Custom View config file.
 
-```js title="constants.js" highlightLines="5-8, 12"
+```js title="constants.js" highlightLines="9"
 import { resolveCustomViewResourceAccesses } from '@commercetools-frontend/application-config';
 
 export const groupNames = {
   delivery: 'delivery',
   promotion: 'promotion',
 };
 
-export const PERMISSIONS = resolveCustomViewResourceAccesses(
-  ['delivery', 'promotion']
+export const PERMISSIONS_WITH_GROUPS = resolveCustomViewResourceAccesses(
+  [groupNames.delivery, groupNames.promotion]
```

Comment:
These values should be specified as string literals so that TS can correctly infer and define the mapping names.

Reply:
Applied in fb7a811b30d8fb02e7b111509af229d054724206